packages/ui-components/src/index.js

```diff
@@ -66,9 +66,9 @@ export { ModalFooter } from "./deprecated_js/ModalFooter/index.js"
 export { NativeSelect } from "./components/NativeSelect/NativeSelect.component"
 export { NativeSelectOption } from "./components/NativeSelectOption/NativeSelectOption.component"
 export { NativeSelectOptionGroup } from "./components/NativeSelectOptionGroup/NativeSelectOptionGroup.component"
-export { Panel } from "./components/Panel/index.js"
-export { PanelBody } from "./components/PanelBody/index.js"
-export { PanelFooter } from "./components/PanelFooter/index.js"
+export { Panel } from "./components/Panel/Panel.component"
```

Comment:
**(Breaking change)**
Panel use Portal, Portal is still in migration, I would suggest to export this component stil from deprecated until we will not finish all of Portal depended components.

Reply:
Very helpful, thanks!